appdx-web3js-tutorial.asciidoc

```diff
@@ -50,21 +50,21 @@ $ <strong>node code/web3js/web3-contract-basic-interaction.js</strong>
 </pre>
 ++++
 
-Use your own Infura token (register at https://infura.io/[] and store the api-key in a local file called __infura_token__):
+Get an access token from a node provider such as https://alchemyapi.io[Alchemy API] or https://infura.io[Infura] and store the api-key in a local file called __alchemyFileToken__ or __infuraFileToken__):  
 
 ++++
 <pre data-type="programlisting">
-$ <strong>node code/web3js/web3-contract-basic-interaction.js \
-  --infuraFileToken /path/to/file/with/infura_token</strong>
+$ <strong>node code/web3js/web3js_demo/web3-contract-basic-interaction.js \
+  --alchemyFileToken /path/to/file/with/alchemy_token</strong>
 </pre>
 ++++
 
 or:
 
 ++++
 <pre data-type="programlisting">
-$ <strong>node code/web3js/web3-contract-basic-interaction.js \
-  /path/to/file/with/infura_token</strong>
+$ <strong>node code/web3js/web3js_demo/web3-contract-basic-interaction.js \
```

Comment:
Was this intended @deric-alchemy ?

Reply:
Yup - the file got moved at some point in the past